selfdrive/controls/lib/latcontrol_indi.py

```diff
@@ -44,11 +45,16 @@ def __init__(self, CP):
     self.sat_limit = CP.steerLimitTimer
 
     self.reset()
+    
+    @property
+  def outer_loop_gain(self):
```

Comment:
Make sure to delete `self.outer_loop_gain = CP.lateralTuning.indi.outerLoopGain` and fix the indentation of property